qa/rpc-tests/bip65-cltv-p2p.py

```diff
@@ -43,7 +43,7 @@ def __init__(self):
     def setup_network(self):
         # Must set the blockversion for this test
         self.nodes = start_nodes(self.num_nodes, self.options.tmpdir,
-                                 extra_args=[['-debug', '-whitelist=127.0.0.1', '-blockversion=3']],
+                                 extra_args=[['-debug', '-whitelist=127.0.0.1', '-blockversion=3', '-buriedsfparams=bip65:1351']],
```

Comment:
This particular change does not change anything in fact, because it is setting the buriefsf height at its default value anyway.
I can't really revert the rest. The reason is that the previous test was testing activation and enforcement. But now we don't have different height for activation and enforcement anymore.